extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaIndexTaskClient.java

```diff
@@ -622,7 +663,7 @@ private FullResponseHolder submitRequest(
             Thread.sleep(sleepTime);
           }
           catch (InterruptedException e2) {
-            Throwables.propagate(e2);
+            throw new RuntimeException(e2);
```

Comment:
This needs to set `Thread.currentThread().interrupt()`

Reply:
Added the interrupt call